.cloudbees/testing/action.yml

```diff
@@ -54,6 +59,14 @@ runs:
       run: |
         set -uo pipefail
 
+        if echo "${{ inputs.chart-location }}" | grep -q '^oci://'; then
+          if [ "${{ inputs.registry-configuration }}" != '' -a -s "${{ inputs.registry-configuration }}" ]; then
+            chart_location=$(echo -n "${{ inputs.chart-location }}" | sed 's|^oci://||')
+            registry-config --config "${{ inputs.registry-configuration }}" resolve "${chart_location}" resolved_registries.out
```

Comment:
To prevent (accidental) shell injection, please assign `${{ inputs.registry-configuration }}` to an `env` var within the yaml (outside the `run` field) and use the env var here. That is return a clear errors for users when they'd attempt to not provide a filepath for that input but the actual JSON configuration or so which would break the script and therefore return a cryptic error.
Same for the `${{ inputs.chart-location }}` (although it is less likely that users would run into a problem with that one being inlined).

Reply:
Done @mgoltzsche !